apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -10173,3 +10181,13 @@ mat-year-view table tr td.mat-calendar-body-label {
     display: none !important;
 }
 /** Angular material datepicker **/
+.pd-0 {
```

Comment:
We already have `p-0` class for setting 0 padding

Reply:
that class is getting overridden if we are using form-control class